xml_inscriptions/transcribed/MA.Glouc.HCM.L.03.8.7.xml

```diff
@@ -60,24 +60,24 @@
                         <!--  *************************************  <objectDesc>  *********************************
                             <objectDesc> contains information about the physical properties of the object and the inscribed text.  -->
                             
-                        <objectDesc ana="#slab">
-                            <supportDesc ana="#clay.terracotta">
+                        <objectDesc ana="#object_other">
+                            <supportDesc ana="#clay">
```

Comment:
Why get rid of terracotta if we're leaving it in prose description?

Reply:
Prof Bodel wants to stick to broader categories, but I think I might have misunderstood how the tagging system works. Because if something is tagged "#clay.terracotta", will it show up if "#clay" is searched?

Reply:
It's supposed to, yes, since it's a faceted search. Any sub-level should show up when you have the higher tier selected -- at least, that's how I recall it being designed.